save-cosv/src/main/kotlin/com/saveourtool/save/cosv/event/CommentListener.kt

```diff
@@ -43,9 +46,14 @@ class CommentListener(
             )
 
         val message = messageNewVulnComment(commentOwner, identifier)
-        recipients.map { userId ->
-            notificationRepository.saveNotification(message, userId)
+        val users = userService.findAllByIdIn(recipients)
```

Comment:
whats the difference between previous variant?

Reply:
only the way, where we iterate? in code or in DB?

Reply:
The first option did not work correctly, it was written through NativeQuery, but now it will work through JPA